v3/lint/registration.go

```diff
@@ -87,6 +92,8 @@ type Registry interface {
 	// WriteJSON writes a description of each registered lint as
 	// a JSON object, one object per line, to the provided writer.
 	WriteJSON(w io.Writer)
+	SetConfiguration(config Configuration)
+	GetConfiguration() Configuration
```

Comment:
As a library consumer of zlint, I just want to say that these new interface methods will work great for us. We [build a registry](https://github.com/letsencrypt/boulder/blob/7dcbf69536eba2d7133067c56e959873f00b4d0b/linter/linter.go#L148-L163) with our own configurable set of skip-lints; adding one more step to load and apply a config from a TOML file will be very easy.